tracim/development.ini.base

```diff
@@ -35,6 +35,7 @@ full_stack = true
 # You can set french as default language by uncommenting next line
 # lang = fr
 cache_dir = %(here)s/data
+preview_cache_dir = /tmp/tracim/cache/previews/
```

Comment:
The file depot directory is not parameterized yet because I have troubles setting it up the same way Alexis setted up its preview generator. this aspect is underway.